internal/pkg/service/buffer/store/configstore/configstore.go

```diff
@@ -124,35 +81,35 @@ func (c *Store) CreateReceiver(ctx context.Context, receiver model.Receiver) (er
 		return err
 	}
 
-	prefix := ReceiverPrefix(receiver.ProjectID)
-	logger.Debugf(`Reading "%s" count`, prefix)
-	allReceivers, err := client.KV.Get(ctx, prefix, etcd.WithPrefix(), etcd.WithCountOnly())
+	prefix := schema.Configs().Receivers().InProject(receiver.ProjectID)
```

Comment:
Nahradil som to v existujucom kode.
Zatial sa to pouziva iba ako generator key/prefix ako string.
V dalsom PR, sa budu nad `Prefix/Key` priamo volat operacie.
Tu by bolo tych zmien uz prilis vela.

Reply:
Ten další PR máš taky nějak rozdělaný?

Jak na to koukám, tak v podstatě jsem tímhle blokovaný, když bych stejnak musel skoro celé předělávat a nebo ty složitě něco rebasovat.

Třeba ten `mapper` bude záviset na tom, že existuje nějaký typ `Receiver` který je "kompletní" - má v sobě i exports a ty exports mají v sobě mapping. Ideálně by byl jeden `store.CreateReceiver` který by přijimal ten "full" `Receiver` typ a volal v sobě i `CreateExport` atd. Pak `mapper` konvertuje `buffer.Receiver` na `model.Receiver` a je to jeden function call. Takže je potřeba upravit i store, ale ten se bude teda měnit na to, aby používal ops.

Reply:
Alebo tu `400`ku. Problem je, ze som tiez tym blokovany.